.spelling

```diff
@@ -8,7 +8,6 @@ Ebuyer
 iMAX
 IntelliJ
 Nikitin
-ODROID
```

Comment:
I suspect there may be some references to the board in the interactive troubleshooter. Not sure if those get checked by the normal spellchecking stuff.

Reply:
Scratch that, grerpping doesn't find anything.

Reply:
I grepped the whole repo for ODROID, almost everything refers to it (correctly) as the Brain.